heft-plugins/heft-sass-plugin/package.json

```diff
@@ -12,10 +12,10 @@
   "types": "dist/heft-sass-plugin.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "heft build --clean",
     "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "build": "heft build --clean",
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
```

Comment:
Is this going to be the suggested way to do phases in Rush+Heft?

Reply:
Yes, I think so. It seems simple enough, and customizable to include multiple Heft phases, if for some reason you want Rush/Heft phases out of sync.